flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalogTable.java

```diff
@@ -65,11 +65,4 @@ public CatalogBaseTable copy() {
 		return Optional.ofNullable(getComment());
 	}
 
-	@Override
-	public Map<String, String> toProperties() {
-		// TODO: output properties that are used to auto-discover TableFactory for Hive tables.
```

Comment:
Since we will not use auto table factory discovery for HiveCatalogTable, we don't need to output anything. In other words, an empty map is sufficient.

We probably should keep this for now, until we combine HiveCatalogTable and GenericCatalogTable.

Reply:
right, I'm actually working on combining the two tables now. Maybe we can just keep this PR as-is? there's not much difference to make it happen sooner than later

Reply:
That's fine as long as you remember the point here. :)